scaleway/resource_lb_frontend.go

```diff
@@ -146,6 +147,11 @@ func resourceScalewayLbFrontend() *schema.Resource {
 											Type: schema.TypeString,
 										},
 									},
+									"http_filter_option": {
+										Type:        schema.TypeString,
```

Comment:
Should we consider any limit for this attribute. Like the length?. Also can this attribute be computed by any other tool like console or API call?

Reply:
It cannot be computed by the console/API call as it is a filter fully defined by the user. As for the length, it depends on each web server but those limits are usually in the order of 4kb to 48kb so I guess it is not worthwhile to add a constraint on the terraform side because we don't control the limit of valid header size accepted on the backend side.